src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -32,14 +32,7 @@
 import java.io.IOException;
 import java.io.UncheckedIOException;
 import java.nio.file.Files;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.NavigableMap;
-import java.util.Set;
-import java.util.TreeMap;
+import java.util.*;
```

Comment:
Normally, I advise against Star import in the production code. It may lead to ambiguous imports in some cases